tests/data/cpms/gw.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This is some rather random data to test against. @rettichschnidi could you maybe provide better examples that we might see during production?

Reply:
Do you intend to use this data against our live backend? Or just for offline use? I'd be a bit hesitating to provide data for the first use case.

Reply:
I do not intend to write integration/system tests.